spec/requests/api/v2/providers/courses/publish_spec.rb

```diff
@@ -140,6 +143,7 @@
             "Enter a course length",
             "Enter details about the salary for this course",
             "Enter GCSE requirements",
+            "The Accredited Body  does not exist in this cycle",
```

Comment:
Rather than checking the presence of this error throughout the spec, it seems like this validation warrants its own context block, asserting that the error appears when the accredited body is not in the current cycle. 

Related comment: https://github.com/DFE-Digital/teacher-training-api/pull/2266/files#r745495169